eclipse/sw360
# feat(SVM): Added new field enableSvm(Default value - true) for linked project.

> Added new field enableSvm(Default value - true) for linked project.
>  * Request/Response structure of get_projects_with_all_details, get_project_by_id, create_project, update_project for linked project changed

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 

### How To Test?
> - Create Moderation Request for Project with Add, remove , update linked project with previous structure as in master.
> - Create Project with linked Project.
> - Run migration script `044_migrate_project_linked_project_relation.py` , `045_migrate_project_moderation_request_linked_project_relation.py`
> - Verify old Project Details, Edit, Delete.
> - Verify old Moderation request View, Accept.
> - Create New Project with/without linked project, test view, edit, delete.
> - Create new Moderation Request for Project with Add, remove , update linked project - verify view, approve.
> - Test Rest API - get_projects_with_all_details, get_project_by_id, create_project, update_project with/without linked project
> - Verify Usage table in detail/Edit page of Project which used by other project(i.e linked in other project)
> - Export excel with/without release from Project List/Details Page - verify linkedProject column
 
> Have you implemented any additional tests? - No

### Checklist
Must:

**_Edit Project-_** 
![1](https://user-images.githubusercontent.com/56534710/124517665-2828de80-de02-11eb-9d2c-5bdd0f024557.JPG)

**_Project Moderation Request-_** 
![2](https://user-images.githubusercontent.com/56534710/124517675-2b23cf00-de02-11eb-931a-1cd1365984ba.JPG)

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
